src/main/java/com/minecolonies/coremod/entity/citizen/EntityCitizen.java

```diff
@@ -167,23 +169,23 @@
     /**
      * IsChild flag
      */
-    private boolean          child               = false;
+    private boolean                child               = false;
     /**
      * Whether the citizen is currently running away
      */
-    private boolean          currentlyFleeing    = false;
+    private boolean                currentlyFleeing    = false;
     /**
      * Timer for the call for help cd.
      */
-    private int              callForHelpCooldown = 0;
+    private int                    callForHelpCooldown = 0;
     /**
      * Distance walked for consuming food
      */
-    private float            lastDistanceWalked  = 0;
+    private float                  lastDistanceWalked  = 0;
     /**
      * Citizen data view.
      */
-    private ICitizenDataView citizenDataView;
+    private ICitizenDataView       citizenDataView;
```

Comment:
Can you add a whitespace between comment and var, so they don't get all moved out?

Reply:
thats how our formatter does it, even if I manually add a whitespace

Reply:
Are you sure? Because when I put whitespace between the fields (between comment and prev field) it won't push them all out.

Reply:
what you're trying to say is you want me to add linebreaks between the variable and the next variables comments, not just a whitespace

Reply:
yes